include/bitcoin/client/obelisk/obelisk_types.hpp

```diff
@@ -48,8 +40,10 @@ typedef std::vector<history_row> history_list;
 
 struct BCC_API stealth_row
 {
-    data_chunk ephemkey;
-    payment_address address;
+    // No sign byte in public key.
+    hash_digest ephemkey;
+    // No version byte in address.
+    short_hash address;
     hash_digest transaction_hash;
```

Comment:
I'm having a hard time connecting this to my understanding of stealth. There would be no reason for the server to store any part of the stealth address. In fact if it could determine anything about the stealth address associated with the transaction there would be a severe problem. The address derived from the tx (along with the ephemeral public key) is an address derived from the stealth secret by converting the stealth public key to a P2PKH address. That makes this address version predictable, regardless of the type of stealth address that was spent towards